be/src/olap/page_cache.cpp

```diff
@@ -21,16 +21,17 @@
 
 #include <ostream>
 
-namespace doris {
-
-StoragePageCache* StoragePageCache::_s_instance = nullptr;
+#include "runtime/exec_env.h"
 
-void StoragePageCache::create_global_cache(size_t capacity, int32_t index_cache_percentage,
-                                           int64_t pk_index_cache_capacity, uint32_t num_shards) {
-    DCHECK(_s_instance == nullptr);
-    static StoragePageCache instance(capacity, index_cache_percentage, pk_index_cache_capacity,
-                                     num_shards);
-    _s_instance = &instance;
+namespace doris {
+StoragePageCache* StoragePageCache::create_global_cache(size_t capacity,
+                                                        int32_t index_cache_percentage,
+                                                        int64_t pk_index_cache_capacity,
+                                                        uint32_t num_shards) {
+    DCHECK(GetGlobalStoragePageCache() == nullptr);
```

Comment:
do not add this dcheck, because we may use this method in unit test, we need call create global cache multi times.